docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,52 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
+
+**Versioned name**: *Round-5*
+
+**Category**: Arithmetic unary operation 
+
+**Short description**: *Round* performs element-wise round operation with given tensor. It takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+* **1**: An tensor of type T. **Required.**
```

Comment:
Surround `T` with asterisk character like this `*T*`

Reply:
Done.